test-requirements.txt

```diff
@@ -20,6 +20,6 @@ pytest-timeout~=1.3.0
 pytest-travis-fold~=1.3.0
 pytest-xdist~=1.15
 requests-mock~=1.2
-pip!=9.0.2, !=10.0.*, !=18.0
+pip!=9.0.2, !=10.0.*, !=18.*
```

Comment:
No.

We dont fix this here.

https://gitlab.com/coala/mobans/blob/master/test-requirements.txt#L23 is where the problem is.

After that is fixed, we sync it to all repos.

Reply:
@nvzard , it has been fixed upstream, can you update this PR accordingly please ?